tests/unit/containers/my-resources/QuestionsContainer.spec.jsx

```diff
@@ -75,17 +89,64 @@ describe('QuestionsContainer test', () => {
     expect(columnLabel).toBeInTheDocument()
     expect(questionTitle).toBeInTheDocument()
   })
+
+  it('should open menu and duplicate a question successfully', async () => {
+    mockAxiosClient
+      .onPost(URLs.resources.questions.post)
+      .reply(200, { success: true })
+
+    expect(screen.getAllByTestId('menu-icon').length).toBeGreaterThan(0)
+
+    const menuButtons = screen.getAllByTestId('menu-icon')
+
+    await waitFor(() => fireEvent.click(menuButtons[0]))
```

Comment:
Is `waitFor` necessary here?
```suggestion
    fireEvent.click(menuButtons[0])
```